e2e/node/e2e.test.ts

```diff
@@ -1774,29 +1774,76 @@ describe(`End-to-end access grant tests for environment [${environment}] `, () =
         );
       });
 
-      it.skip("can iterate through pages", async () => {
+      it("can iterate through pages", async () => {
         const pages = paginatedQuery(
           {
-            pageSize: 20,
+            pageSize: 2,
             type: "SolidAccessRequest",
+            issuedWithin: "P1D",
           },
           {
             fetch: addUserAgent(requestorSession.fetch, TEST_USER_AGENT),
             // FIXME add query endpoint discovery check.
             queryEndpoint: new URL("query", vcProvider),
           },
         );
-        const maxPages = 2;
-        let pageCount = 0;
         for await (const page of pages) {
           expect(page.items).not.toHaveLength(0);
-          pageCount += 1;
-          // Avoid iterating for too long when there are a lot of results.
-          if (pageCount === maxPages) {
+        }
+      }, 120_000);
+
+      it("shows updated status for an approved request", async () => {
+        // Issue an Access Request.
+        const request = await issueAccessRequest(
+          {
+            access: { read: true },
+            resourceOwner: resourceOwnerSession.info.webId as string,
+            resources: [sharedFileIri],
+            expirationDate: new Date(Date.now() + 60 * 60 * 1000),
+          },
+          {
+            fetch: addUserAgent(requestorSession.fetch, TEST_USER_AGENT),
+            accessEndpoint: vcProvider,
+            updateAcr: false,
+            returnLegacyJsonld: false,
+          },
+        );
+        console.debug("Access request issued");
```

Comment:
Do you need this debug log?

Reply:
It was forgotten when cleaning up the tests debug, removed in c573ab0cca099201238f5effb04236cc35478892.